app/views/posts/index.html.erb

```diff
@@ -1,2 +1,22 @@
-<h2>Post#index</h2>
-<p>Here are ALL Posts for a given user you are in '/users/user_id/posts'</p>
+<% content_for :title, "#{@user.name}'s Posts " %>
+<%= render "users/user", user: @user %>
+<% @user.posts.each do | post | %>
+  <%= link_to user_post_path(@user, post) do %>
+    <%= render "post", post: post %>
+    <div class="user-post-comments mt-2">
+      <h3 class="text-xl">
+        <span class="underline decoration-wavy decoration-slate-500">Comments</span>
+        <span>For Post: </span>
+        <span class="text-blue-gray-600 underline decoration-red-400"><%= "#{post.title}"%></span>
+      </h3>
+      <% post.recent_five.each do |comment| %>
+        <div class='post-comment'>
+          <p>
+            <span><strong><%= comment.author.name %></strong></span>
+            <span><%= comment.text %></span>
+          </p>
+        </div>
+      <%end%>
+    </div>
```

Comment:
- [ ] Please fix[ this issue](https://github.com/RashidMenhas/Rails_Blog_App/pull/7#discussion_r1158156703) raised by the previous reviewer.